pages/common/ld.md

```diff
@@ -0,0 +1,16 @@
+# ld
+
+> The GNU ELF linker.
+> More information: <https://ftp.gnu.org/old-gnu/Manuals/ld-2.9.1/html_mono/ld.html>.
+
+- "Link" an object file with no dependencies into an executable:
+
+`ld {{file.o}} -o {{executable}}`
```

Comment:
Thank you, that makes sense. In the last example I have paths to a few common linux libraries, 1) should I avoid those because this is in common/ and if not 2) should I keep the common paths or should I replace them with `{{path/to/...}}`s if that makes sense?

Reply:
If the example is specific for `glibc` then it's OK.